app-text/asciidoc/asciidoc-9.0.0_rc1.ebuild

```diff
@@ -66,7 +66,6 @@ src_configure() {
 
 src_test() {
 	local -x ASCIIDOC_PY=asciidoc.py
-	"${EPYTHON}" tests/test${PN}.py update || die
```

Comment:
Please squash into the original version bump.

Reply:
OK